### ISSUE TITLE
[Docs] Publish information about contributing to Meshery Docs

### ISSUE THREAD

**Issue by leecalcote** [layer5io/meshery#3011]
#### Current State
Meshery Docs has contributing information in the [`/docs` folder's readme](https://github.com/layer5io/meshery/tree/master/docs). There is a new contributing section in docs.meshery.io that would benefit from having a dedicated page to contributing to docs.

#### Desired State

1. create a new page: `projects/meshery-docs`; use `projects/meshery-ui` as a reference.
1. create an entry in toc.yaml for this new page.
1. copy the contents of the `/docs` readme.
1. include a link to the [Meshery Documentation design specification](https://docs.google.com/document/d/17guuaxb0xsfutBCzyj2CT6OZiFnMu9w4PzoILXhRXSo/edit).
1. include [instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

---

#### Contributor Resources

- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
- [Layer5 Community Slack](http://slack.layer5.io)

**Comment by ricknjacky** [comment]
Can I take up this issue?

**Comment by leecalcote** [comment]
@ricknjacky that'd be great. 👍

**Comment by ricknjacky** [comment]
> @ricknjacky that'd be great. 👍

Thanks, any heads up on how to obviate this? I am quite new to the codebase...any insight would be really helpful.

**Comment by leecalcote** [comment]
@ricknjacky, yes, go through the instructions and links in the issue description. Ask specific questions as they arise.

**Comment by ricknjacky** [comment]
Hey @leecalcote I've read the resources they're really helpful. I am sanguine that I'll be able to fix this as soon as I get my query answered.

> #### Desired State
> 1. create a new page: `projects/meshery-docs`; use `projects/meshery-ui` as a reference.

Where is the  `projects/meshery-ui` source-code? Because I found [project/contributing-ui](https://docs.meshery.io/project/contributing-ui) page. Pardon me if this was too naïve, looking forward to some insights on this, Thanks in advance.

**Comment by leecalcote** [comment]
+ @adithyaakrishna

**Comment by leecalcote** [comment]
@ricknjacky the `projects/meshery-ui` file is in the same directory as the `project/contributing-ui` file.

I spent a little bit of time today updating quite a few docs, including an initial guide on contributing to the Meshery docs. There's more to write up, but his covers the majority of what was needed. If I don't see an update from you by the Meshery Development meeting this week, I'll assume that we're covered here and will close out the issue.

**Comment by ricknjacky** [comment]
hey I am sorry, caught up with something else as of now, would love to comeback at meshery soon, hope you understand.

**Comment by ankitzm** [comment]
Can you please assign me this issue.

**Comment by adithyaakrishna** [comment]
Yes sure, go ahead 🎉

**Comment by debjani1413** [comment]
Is this issue been fixed already ???

**Comment by adithyaakrishna** [comment]
@debjani1413 I don't think so as it does not have any Linked PRs. If you don't mind, could you please check regarding this?
If it's not yet fixed, would you be willing to take this up? 👀

**Comment by debjani1413** [comment]
@adithyaakrishna yes, sure I would love to take it up!

**Comment by debjani1413** [comment]
@adithyaakrishna I guess it has been worked upon because I see a completely dedicated page for contributing to the docs which was the purpose of this issue!!

**Comment by debjani1413** [comment]
Hey @leecalcote I can see a page that is fully dedicated to, the guidelines for contributing to meshery docs, was not that the purpose of this issue?? Or did I misinterpret it 😐

**Comment by leecalcote** [comment]
@debjani1413 I'm behind on the thread here. Will you list this as an agenda item in this week's Meshery Development Meeting so that clarity can be offered verbally?

**Comment by ankitzm** [comment]
I am unable to work due to various reasons, so I have unassigned myself so that the issue can be resolved by someone else

**Comment by debjani1413** [comment]
@leecalcote I guess I may have missed this one out, do you still mind me working at it???

**Comment by adithyaakrishna** [comment]
@debjani1413 Sure you can go ahead :)

**Comment by Shatakshi0805** [comment]
Can I work on this?

**Comment by warunicorn19** [comment]
@debjani1413 are you working on this? Or can @Shatakshi0805 go ahead with it?

**Comment by leecalcote** [comment]
@Shatakshi0805, yes, help is needed. Are you still up to the task?

**Comment by vedant-kakde** [comment]
@leecalcote @adithyaakrishna  I would like to work on this issue.
